docs/CHANGELOG.rst

```diff
@@ -128,6 +128,8 @@ Changed
 
   - ``numpy`` (v1.14.4)
   - ``pandas`` (v0.23.0)
+  - remove ``bedgraphtobigwig`` and add ``deepTools`` to ``resolwebio/rnaseq:3.4.0``
```

Comment:
Wrong indentation. Add `deepTools` version. Consider establishing a relationship between the two tools, e.g. _replace `bedgraphtobigwig` with `deepTools`_ and provide a brief explanation why this is needed.